TACC/Core-Styles
# refactor: simpler global message links styles

## Overview

Simplify syntax for global message link styles.

## Changes

- **change** `x-message` to use `:is()`

## Testing

1. Open any messages demo.
2. Verify global message link UI is unchanged.
2. Verify global message link UI on state change is unchanged.

## UI

<img width="920" alt="global message styles unchanged" src="https://github.com/user-attachments/assets/2b99e56c-8610-440a-b0fe-969fc1309aea">

https://github.com/user-attachments/assets/60883b65-6947-431d-a364-a31f32d1227c
